tests/driver_enc28j60/main.c

```diff
@@ -11,36 +11,58 @@
  * @{
  *
  * @file
- * @brief       Test application for the ENC28J60 Ethernet device driver
+ * @brief       Test application for ENC28J60 ethernet device driver
  *
- * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ * @author      Leandro Lanzieri <leandro.lanzieri@haw-hamburg.de>
  *
  * @}
  */
 
 #include <stdio.h>
 
 #include "shell.h"
-#include "msg.h"
+#include "assert.h"
+#include "enc28j60.h"
+#include "enc28j60_params.h"
+#include "test_utils/netdev_eth_minimal.h"
+#include "init_dev.h"
+
+static enc28j60_t enc28j60[ENC28J60_NUM];
+
+int netdev_eth_minimal_init_devs(netdev_event_cb_t cb) {
+    for (unsigned i = 0; i < ENC28J60_NUM; i++) {
+        netdev_t *device = &enc28j60[i].netdev;
+
+        /* setup the specific driver */
+        enc28j60_setup(&enc28j60[i], &enc28j60_params[i], i);
+
+        /* set the application-provided callback */
+        device->event_callback = cb;
+
+        /* initialize the device driver */
+        int res = device->driver->init(device);
+        assert(!res);
+    }
+
+    return 0;
+}
 
-#define MAIN_QUEUE_SIZE     (8U)
-static msg_t _main_msg_queue[MAIN_QUEUE_SIZE];
```

Comment:
Could you also drop the second empty line to please the CI?

Reply:
Fixed